cloudsupport/cloudproviderconfiguration.go

```diff
@@ -145,8 +145,19 @@ func GetListEntitiesForPoliciesFromCloudProvider(cluster string, cloudProvider s
 		//TODO - implement GKE support
 		return nil, fmt.Errorf(cloudsupportv1.NotSupportedMsg)
 	case cloudsupportv1.AKS:
-		//TODO - implement AKS support
-		return nil, fmt.Errorf(cloudsupportv1.NotSupportedMsg)
+		aksSupport := cloudsupportv1.NewAKSSupport()
```

Comment:
Can we add unitests?

Reply:
I added unitests for those we have mock interfaces, this func create a "real" AKSSupport obj which tries getting resources from a cluster so I think we can't here